hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/evm/contracts/operations/SelfDestructOperationTest.java

```diff
@@ -60,4 +103,69 @@ void testExecuteWithInvalidOwner() {
                                 () -> contractCallService.processCall(serviceParameters))
                         .getMessage());
     }
+
+    private void feeSchedulesPersist() {
+        CurrentAndNextFeeSchedule feeSchedules = CurrentAndNextFeeSchedule.newBuilder()
+                .setCurrentFeeSchedule(FeeSchedule.newBuilder()
+                        .addTransactionFeeSchedule(TransactionFeeSchedule.newBuilder()
+                                .setHederaFunctionality(EthereumTransaction)
+                                .addFees(FeeData.newBuilder()
+                                        .setServicedata(FeeComponents.newBuilder()
+                                                .setGas(852000)
+                                                .build()))))
+                .build();
+        domainBuilder
+                .fileData()
+                .customize(f -> f.fileData(feeSchedules.toByteArray()).entityId(FEE_SCHEDULE_ENTITY_ID))
+                .persist();
+    }
+
+    private void exchangeRatesPersist() {
+        final ExchangeRateSet exchangeRatesSet = ExchangeRateSet.newBuilder()
+                .setNextRate(ExchangeRate.newBuilder()
+                        .setCentEquiv(15)
+                        .setHbarEquiv(1)
+                        .build())
+                .build();
+        domainBuilder
+                .fileData()
+                .customize(f -> f.fileData(exchangeRatesSet.toByteArray()).entityId(EXCHANGE_RATE_ENTITY_ID))
+                .persist();
+    }
+
+    private Long selfDestructContractPersist() {
+        final var selfDestructContractBytes = functionEncodeDecoder.getContractBytes(selfDestructContractPath);
+        final var selfDestructContractEntity = domainBuilder.entity().persist();
+
+        domainBuilder
+                .contract()
+                .customize(c -> c.id(selfDestructContractEntity.toEntityId().getId())
+                        .runtimeBytecode(selfDestructContractBytes))
+                .persist();
+
+        domainBuilder
+                .recordFile()
+                .customize(f -> f.bytes(selfDestructContractBytes))
+                .persist();
+
+        return selfDestructContractEntity.getNum();
```

Comment:
Shouldn't use num directly.

```suggestion
        return selfDestructContractEntity.toEntityId();
```

Reply:
Thanks, I've applied your suggestion